src/site/content/en/lighthouse-accessibility/document-title/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Ensure that each HTML document contains a title
+title: Ensure that each HTML document contains a `<title>`
```

Comment:
This should be left as-is since _title_ isn't a literal here. And, anyway, this post title is being updated in the lighthouse-a11y cleanup PR.